Example/src/screens/SimpleNativeStack.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {View, StyleSheet} from 'react-native';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationProp,
+} from 'react-native-screens/native-stack';
+import {Button} from '../shared';
+
+type StackParamList = {
+  Main: undefined;
+  Detail: undefined;
+};
+
+interface MainScreenProps {
+  navigation: NativeStackNavigationProp<StackParamList, 'Main'>;
+}
+
+const MainScreen = ({navigation}: MainScreenProps): JSX.Element => (
+  <View style={styles.container}>
+    <Button
+      title="Go to detail"
+      onPress={() => navigation.navigate('Detail')}
+    />
+    <Button onPress={() => navigation.pop()} title="🔙 Back to Examples" />
+  </View>
+);
+
+interface DetailScreenProps {
+  navigation: NativeStackNavigationProp<StackParamList, 'Detail'>;
+}
+
+const DetailScreen = ({navigation}: DetailScreenProps): JSX.Element => (
+  <View style={styles.container}>
```

Comment:
Add some colors in the screens with transitions to better see them.

Reply:
Fixed in https://github.com/software-mansion/react-native-screens/pull/827/commits/877fddaed2955733f371f833199f5456b9198bee